src/frontend/flask_port.py

```diff
@@ -25,6 +25,14 @@ def homepage():
     )
 
 
+@app.route("/info.html")
+def info_page():
+    """Route to info page."""
+    print(f"Session on info page: {session}")  # Debug log
```

Comment:
Remove print statement, use logger (https://docs.python.org/3/library/logging.html) if output is wanted.